test/grabURITest.coffee

```diff
@@ -0,0 +1,222 @@
+# Copyright (C) 2016 The Firehose Project
+#
+# This software may be modified and distributed under the terms
+# of the MIT license.  See the LICENSE file for details.
```

Comment:
Note how it's short and sweet. Screw the whitespace and extra lines, the shorter the better. You need to enforce copyright and license, everything else is bust.